go-controller/pkg/ovn/ovn.go

```diff
@@ -202,8 +202,8 @@ func (oc *Controller) Run() error {
 		return err
 	}
 
-	for _, f := range []func() error{oc.WatchPods, oc.WatchServices, oc.WatchEndpoints,
-		oc.WatchNamespaces, oc.WatchNetworkPolicy} {
+	for _, f := range []func() error{oc.WatchPods, oc.WatchNamespaces, oc.WatchServices,
```

Comment:
I think this should actually be:
oc.WatchNamespaces, oc.WatchPods, oc.WatchServices, oc.WatchEndpoints, oc.WatchNetworkPolicy

Reply:
@trozet Changes made, PTAL.